docs/modpacks/play.md

```diff
@@ -63,3 +63,21 @@ Click the "Add a new Minecraft Core or Mod Pack" tile in the Start Menu, then cl
 ![Add a new Minecraft Core or Mod Pack Tile](../../static/img/tutorial/BakaXLAddCoreTile.png)
 
 Click "Continue" and drag the modpack file into BakaXL.
+
+### Superpack
+
+Superpack is an application for extracting/downloading modpacks to directories.
+
+#### Install through Superpack
+
+Modpacks can be installed by simply navigating or searching the required modpack.
+
+![Superpack homepage](../../static/img/tutorial/superpackHome.png)
+
+Click on the desired modpack to download and view details!
+
+![Superpack modpack page](../../static/img/tutorial/superpackSearch.jpg)
+
+In case you want to use ID or slug of the modpack to view it, click on the "Jump to Project" button on the top-right.
+
+Also, in case of manual file imports, just switch to "Import from file" page.
```

Comment:
See above comment about splitting this up into a separate section. This is also fairly confusing to read; I would word it more like this:

> If you have a project ID or slug, click on the "Jump to Project" button and insert it into the text box. Alternatively, if you have a direct `.mrpack` file, you can go to "Import from file" on the left side.